README.md

```diff
@@ -165,6 +165,11 @@ run('http-server .', {async: true}).then((output) => {
 }).catch((error) => {
   throw error
 })
+run('webpack', {asyncSpawn: true, args: ['--progress', '-p']}).then((output) => {
```

Comment:
I think we could use `spawn` under `async` option. Leaving `exec` under `async` does not remove the issue with the buffer so the bug will still exist. The thing which we need to consider though is to keep the same interface for the user (how `run` with async option is called, handling same options as sync and returning the command outcome).

Important interface requirements:
- user should be able to write whole command with args as a string: `run('ls -la', {async: true})` without giving args separately
- handling same options as for sync (timeout option is an issue for `spawn` but it can be easily simulated with `setTimeout` and `childProcess.kill`)
- `run` should return the outcome of the command even for `async` option. For `exec` that was easy because the content of the buffer was returned` for `spawn` it's an issue. We can create kind of a `LeakyBuffer` which would store let say last 512 kb of data and return that.